examples/routing_service/README.md

```diff
@@ -0,0 +1,16 @@
+# Routing Service
+
+RTI Routing Service is an out-of-the-box solution for integrating disparate and
```

Comment:
Modify as follows:

RTI Routing Service is an out-of-the-box solution that allows developers to rapidly scale and integrate real-time systems that are disparate or geographically dispersed. Routing Service can be used to bridge DDS applications across different domains to support a systems-of-systems architecture, interoperate new and legacy applications to support new product deployment while maintaining previous generation products or to integrate independently-developed systems for progressive system upgrades. 

The Routing Service Adapter SDK allows you to build gateways for integrating Connext DDS with other connectivity technologies. The Adapter SDK enables you to bridge multiple networks and topologies, and to form more complex topologies, as needed; truly realizing the vision of Industrial Internet of Things.

In this directory you will find some examples illustrating the most important features of RTI Routing Service, including support for custom Adapters and Processors. For more information, please refer to the [RTI Routing Service User's Manual](link...)